app/src/main/java/org/covidwatch/android/CovidWatchFirebaseAnalytics.kt

```diff
@@ -0,0 +1,41 @@
+package org.covidwatch.android
+
+import com.google.firebase.iid.FirebaseInstanceId
+import com.google.firebase.analytics.FirebaseAnalytics
+import android.os.Bundle
+
+/*
+This is not a class because we need a "static" variable to hold the isTesting value
+ */
+
+var isTesting: Boolean = false
+var firebaseAnalytics: FirebaseAnalytics? = null
+
+fun getFirebaseId(): String {
+    val instanceId = FirebaseInstanceId.getInstance().id
+    return instanceId
+}
+
+fun setTester(testing: Boolean) {
+    isTesting = testing
+}
+
+fun setAnalyticsInstance(instance: FirebaseAnalytics){
+    firebaseAnalytics = instance
+}
+
+fun sendEvent(name: String){
+    if (isTesting == false) return
+    val bundle = Bundle()
+    /*
```

Comment:
Is there a reason we're opting for the string directly in the code `"item_id"` instead of the constants?

Reply:
"event_date" is now in the code because there is no constant for it